recipes/mne-python/construct.yaml

```diff
@@ -51,177 +51,74 @@ menu_packages:
 
 channels:
   - conda-forge
-  # TODO: ⛔️ ⛔️ ⛔️ DEV BUILDS START: COMMENT OUT BEFORE RELEASE! ⛔️ ⛔️ ⛔️
-  # - conda-forge/label/mne_dev
-  # - conda-forge/label/mne-bids_dev
-  # TODO: ⛔️ ⛔️ ⛔️ DEV BUILDS STOP: COMMENT OUT BEFORE RELEASE! ⛔️ ⛔️ ⛔️
 specs:
   # Python
   - python =3.13.3  # [not (osx and arm64)]
   - python =3.13.2  # [osx and arm64]  # allow_outdated
   - pip =25.1.1
   - wheel =0.45.1
   - conda =25.3.1
-  - mamba =2.1.0
-  # MNE ecosystem
-  # TODO: ⛔️ ⛔️ ⛔️ DEV BUILDS START: CHANGE BEFORE RELEASE! ⛔️ ⛔️ ⛔️
-  # - mne-base =1.4dev0=*_20230503
-  # - mne-installer-menus =1.4dev0=*_20230503
-  # - mne-bids =0.11dev0=*_20221007
-  # TODO: ⛔️ ⛔️ ⛔️ DEV BUILDS STOP: CHANGE BEFORE RELEASE! ⛔️ ⛔️ ⛔️
-  - mne =1.9.0=pyside6_*_100
-  - mne-installer-menus =1.9.0=*_100
```

Comment:
This package is still referenced [here](https://github.com/scientific-python/installer/blob/ed89291becce1ab9a65dda3a5cf2e827cc82baec/recipes/mne-python/construct.yaml#L49-L50)